test/runtests.jl

```diff
@@ -221,14 +223,50 @@ end
         neighbour_finder=DistanceNeighbourFinder(nb_matrix, 10, 1.2f0),
         thermostat=AndersenThermostat(10.0f0),
         loggers=Dict("temp" => TemperatureLogger(Float32, 10),
-                        "coords" => CoordinateLogger(Float32, 10)),
+                        "coords" => CoordinateLogger(Float32, 10),
+                        "energy" => EnergyLogger(Float32, 10)),
         timestep=timestep,
         n_steps=n_steps
     )
 
     @time simulate!(s, parallel=false)
 end
 
+@testset "General interactions" begin
+    n_atoms = 100
+    general_inter_types = (
+        LennardJones(true ), SoftSphere(true ), Mie(5, 10, true ), Coulomb(true ), Gravity(10.0, true ),
+        LennardJones(false), SoftSphere(false), Mie(5, 10, false), Coulomb(false), Gravity(10.0, false),
+    )
+
+    for gi in general_inter_types
```

Comment:
You could add a `@testset "$gi"` in front of the `for` in order to be easier to tell which test fails.